Lectures/Lecture2_GitforCollaboration.md

```diff
@@ -29,10 +29,14 @@ The (physics) research community is the main community that should be considered
 + Readme, templates, community rules, licensing and credit
 
 ## Evaluation
-The students will go through lessons 6-13 of the [git tutorial](https://git.science.uu.nl/os4p-2023/git-tutorial) and will be asked to complete an exercise to demonstrate their understanding of the material.
+During the four hours of tutorial, the students will go through the thirteen lessons of the [git tutorial](https://git.science.uu.nl/os4p-2023/git-tutorial) and will be asked to complete an exercise to demonstrate their understanding of the material.
 
 ### Assignment description
-Students will be grouped in threes and asked to create a repository on GitLab and to use it to collaborate with the two others on a small project where they write a [haiku poem](https://en.wikipedia.org/wiki/Haiku) about Open Science, each contributing one line.
+Students will form teams of threes and asked to create a repository on GitLab and to use it to collaborate with the two others on a small project where they write a [haiku poem](https://en.wikipedia.org/wiki/Haiku) about Open Science, each contributing one line from their own branch. They then create a Merge Request to merge the file with the haiku into the master branch. The students will use the features of GitLab to manage this assignment.
```

Comment:
Should the students create their own repository or their own branch on the course repository? In the assignment it mentions creating a branch of the course repository and here it mentions they should create their own repository.

Reply:
I meant a branch; good catch! ALthough we could also choose for the groups to fork the repository?

Reply:
Yes, that is also possible. Although that means they need to merge back the changes from their fork to the main repository. Maybe this will complicate things a bit too much?

Reply:
I don't think 'complication' is a criterium for not doing it. We want the students to go deep and challenge them?

Reply:
Yes, that's true. I still think that branching is the more important one to practice first, because we will also use that later on for other assignments. But yes, forking is also a possibility